lib/src/main/java/coppercore/utils/FieldFinder.java

```diff
@@ -0,0 +1,169 @@
+package coppercore.utils;
```

Comment:
This can go in as-is for now, but could you create a ticket to generalize this logic please

Reply:
yes I was planning on updating that and a few others such as the Field Constants (as it basically all relates to this year specifically )